diktat-test-framework/src/main/kotlin/org/cqfn/diktat/test/framework/processing/TestProcessingFactory.kt

```diff
@@ -80,4 +76,8 @@ class TestProcessingFactory(private val argReader: TestArgumentsReader) {
         return test.initTestProcessor(testConfig, argReader.properties)
                 .runTest()
     }
+
+    companion object {
```

Comment:
do we check that companion object is last in the class?

Reply:
Yes, in `ClassLikeStructuresOrderRule`

Reply:
good thanks